WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -206,7 +208,8 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleDoubleTapCoordinate:(FBRouteRequest *)request
 {
-  XCUICoordinate *doubleTapCoordinate = [self.class getGestureCoordinate:request];
+  CGPoint doubleTapPoint = CGPointMake((CGFloat)[request.arguments[@"x"] doubleValue], (CGFloat)[request.arguments[@"y"] doubleValue]);
+  XCUICoordinate *doubleTapCoordinate = [self.class gestureCoordinateWithCoordinate:doubleTapPoint application:request.session.application shouldApplyOrientationWorkaround:YES];
```

Comment:
> 1. On iOS 9.x coordinates are not inverted

Yes, but only coordinates we get when we want to know XCUIElement frame. All the other XCUIElement methods have the same bug as in 10.0+

> On iOS 10.x coordinates are inverted

Yes, but this might be fixed in any of upcoming iOS releases.

> 3. We can assume that coordinates passed by user are correct (meaning they are in reference to current orientation). This is mainly because we give them the coordinates or they manually calculate them.

I want to discuss this point with @imurchie, because most of WDA calls where we do operations with absolute screen coordinates cannot be invoked directly from Appium, but rather via TouchAction interface. And we need to be in sync about our expectations here.